### PR TITLE
Forward spot trade events for okex

### PR DESCRIPTION
**Pull request by yurikoval**
This will forward incoming trade data for spot

### CODE REVIEW

File: apps/tai/test/tai/venue_adapters/okex/stream/trades_test.exs [hunk 1]
```diff
@@ -10,25 +10,30 @@ defmodule Tai.VenueAdapters.OkEx.Stream.TradeTest do
...
-        "side" => "buy",
-        "trade_id" => venue_trade_id,
-        "price" => "5556.91",
-        "qty" => "5",
-        "instrument_id" => "BTC-USD-190628",
-        "timestamp" => "2019-05-06T07:19:37.496Z"
-      }
-    ]
+      venue_trades = [
+        %{
```
Comment (rupurt) [inline apps/tai/test/tai/venue_adapters/okex/stream/trades_test.exs]: Is this the same message format for spot? Specifically the `instrument_id`. I've found that the messages can often be quite different.

Comment (yurikoval) [reply apps/tai/test/tai/venue_adapters/okex/stream/trades_test.exs]: Here's an sample of that unhandled message:

```json
{
  "data": {
    "msg": {
      "data": [
        {
          "instrument_id": "ADA-BTC",
          "price": "0.00000497",
          "side": "sell",
          "size": "140",
          "timestamp": "2019-09-18T04:05:18.619Z",
          "trade_id": "67497327"
        }
      ],
      "table": "spot/trade"
    },
    "received_at": "2019-09-18T04:05:18.658538Z",
    "venue_id": "okex"
  },
  "type": "Tai.StreamMessageUnhandled"
}
```
`instrument_id` looks correct, but there might be a difference in `size` -- might be an error in the test.

Comment (yurikoval) [reply apps/tai/test/tai/venue_adapters/okex/stream/trades_test.exs]: I added 3 separate test for each market types

**Comment by rupurt** [comment]
@yurikoval can you also add a test for the event e.g. https://github.com/fremantle-capital/tai/blob/master/apps/tai/test/tai/events/order_update_invalid_status_test.exs#L4

File: apps/tai/test/tai/events/trade_test.exs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        side: :buy,
+        venue_trade_id: venue_trade_id
+      })
+
+    assert %{} = json = Tai.LogEvent.to_data(event)
+    assert json.venue_id == :my_venue
+    assert json.symbol == :btc_usd
+    assert json.received_at == received_at
+    assert json.timestamp == timestamp
+    assert json.price == Decimal.cast(1000.5)
```
Comment (rupurt) [inline apps/tai/test/tai/events/trade_test.exs]: These will need to be strings to be serialized to json

Comment (yurikoval) [reply apps/tai/test/tai/events/trade_test.exs]: Added a test to verify that current behaviour serialises values to strings: https://github.com/fremantle-capital/tai/pull/66/commits/07d70337f695a05e19ff52d1316ef156c21c9350

**Pull request merged**